entity-management-web/src/main/java/eu/europeana/entitymanagement/web/service/EntityRecordService.java

```diff
@@ -292,11 +292,12 @@ public EntityRecord createEntityFromMigrationRequest(
     entityRecord.setEntity(entity);
 
     europeanaProxyEntity.setEntityId(entityId);
-    setEuropeanaMetadata(europeanaProxyEntity, entityId, entityRecord, timestamp);
 
     // create metis Entity
     Entity metisEntity = EntityObjectFactory.createProxyEntityObject(type);
 
+    // set proxies
+    setEuropeanaMetadata(europeanaProxyEntity, entityId, entityRecord, metisEntity, timestamp);
```

Comment:
metis entity is not needed, eventually the id of the metisEntity needs to be added if it is not allready included. However the migration method is obsoleted and it is not used anymore, the this functionality needs to be added to the create method